tests/integration/test_charm.py

```diff
@@ -209,27 +216,32 @@ async def test_dashboard_password_rotation(ops_test: OpsTest):
 
 @pytest.mark.group(1)
 @pytest.mark.abort_on_fail
-async def test_local_password_rotation(ops_test: OpsTest):
-    """Test password rotation for local users -- in case we decide to have any."""
-    user = "monitor"
-    password = await get_user_password(ops_test, user)
-    assert len(password) == 32
-
-    leader_num = await get_leader_id(ops_test)
-
-    # Change both passwords
-    result = await set_password(ops_test, username=user, num_unit=leader_num)
-    assert f"{user}-password" in result.keys()
-
+async def test_cos_relations(ops_test: OpsTest):
```

Comment:
Missing the same test as we have on `opensearch`, where you `curl` directly the `/metrics` endpoints and see if there is data coming out.

Reply:
Note: this request introduces a dependency to the PR.

We can only implement that test after:
 - we have the snap deploying the exporter service
 - that snap has been released (note that we have major issues around dashboards snap releases)
 - that snap has been taken into the Dashboards code.

Fine with me, we can delay this code to that point. Alternatively we can have quick small PRs avoiding too much to pile up for last minute.

Reply:
Linked it against https://warthogs.atlassian.net/browse/DPE-4879 being 'blocked by', and moved the ticket to a BLOCKED state